gui-tk/wiring.tcl

```diff
@@ -559,10 +559,23 @@ proc doubleButton {x y} {
 }
 # for ticket 1062, new hierarchy of context menu access on mouse right click: wires, items and background canvas.
 bind .wiring.canvas <<contextMenu>> {
+	set item minsky.canvas.item
     if [getWireAt %x %y] {
-        wireContextMenu %X %Y  	
+		# if not display the contents of groups, disable context menu of wires. for ticket 1225.
+		if [getItemAt %x %y] {
```

Comment:
The idea is right, but wires have a "visible" attribute that more straightforwardly solves this problem.